sda-auth/dev-server/docker-compose.yml

```
@@ -43,7 +43,7 @@ services:
     build:
       context: ../
       args:
-        GOLANG_VERSION: ${GOLANG_VERSION-1.18}
+        GOLANG_VERSION: ${GOLANG_VERSION-1.20.5}
```

```suggestion
        GOLANG_VERSION: ${GOLANG_VERSION-1.20}
```

No reason to overspecify here, and if anything, the latest version should be used.

---

(Latest 1.20, i.e. 1.20.7 today.)